obsplus/utils/stations.py

```diff
@@ -38,15 +49,42 @@ def df_to_inventory(df) -> obspy.Inventory:
     df
         A dataframe which must have the same columns as the once produced by
         :func:`obsplus.stations_to_df`.
+    client
+
 
     Notes
     -----
-    The dataframe can also contain columns named "sensor_keys" and
-    "datalogger_keys" which will indicate the response information should
-    be fetched suing obspy's ability to interact with the nominal response
-    library. Each of these columns should either contain tuples or strings
-    where the keys are separated by double underscores (__).
+    There are two ways to include response information:
+
+    1. Using the Nominal Response Library (NRL):
+    (https://docs.obspy.org/master/packages/obspy.clients.nrl.html)
+    If the dataframe has columns named "sensor_keys" and "datalogger_keys"
+    these will indicate the response information should
+    be fetched using ObsPy's NRL client. Each of these columns should either
+    contain tuples or strings where the keys are separated by double
+    underscores (__).
+    For example, to specify sensor keys:
+    ('Nanometrics', 'Trillium 120 Horizon')
+    or
+    'Nanometrics__Trillium 120 Horizon'
+    are both valid.
+
+    2. Using a station client:
+    If the dataframe contains a column get_stations_kwargs it indicates that
+    either a client was passed as the client argument or the fdsn IRIS client
+    should be used to download station information. The contents of this column
+    must be a dictionary of acceptable keyword arguments for the client's
+    `get_stations` method. All time values must be provided as iso8601
+    strings.
+    For example,
+    {'network': 'UU', 'station': 'TMU', 'location': '01', 'channel': 'HHZ',
+     'starttime': '2017-01-02',}
+    would use either a provided station client (or ObsPy's default FDSN client
```

Comment:
Would it be more concise to say "would be passed to the station client"? You already explain the choices for the client earlier.